x-pack/plugins/stack_connectors/server/connector_types/jira/schema.ts

```diff
@@ -42,6 +42,7 @@ export const ExecutorSubActionPushParamsSchema = schema.object({
       )
     ),
     parent: schema.nullable(schema.string()),
+    customFields: schema.nullable(schema.recordOf(schema.string(), schema.string())),
```

Comment:
Are we sure that the value of a customField has to be string? Couldn't it be integer or boolean?

Reply:
That's a very good question. When calling the [get create issue metadata API](https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issues/#api-rest-api-2-issue-createmeta-projectidorkey-issuetypes-issuetypeid-get) there were some exotic types but surprisingly no `boolean` or `int`. There might be but I'd rather be conservative in the beginning 😅

Reply:
I can see users wanting to put more value types in the object. As we cannot predict the evolvement of Jira's API and we are not confident that `boolean` or `int` is not supported maybe we should allow any type and let Jira fail on a malformed field. It is a trade-off for sure.

Reply:
> Should we limit the number of custom fields they can pass?

10 like in cases?